packages/teams-js/src/public/calendar.ts

```diff
@@ -54,15 +65,24 @@ export namespace calendar {
     return ensureInitialized(runtime) && runtime.supports.calendar ? true : false;
   }
 
+  /** Open calendar item parameters. */
   export interface OpenCalendarItemParams {
+    /** A string representing the unique ID of the calendar item to be opened. */
     itemId: string;
   }
 
+  /** Compose meeting parameters */
+
   export interface ComposeMeetingParams {
+    /** An array of email addresses of the attendees to invite to the meeting. */
     attendees?: string[];
+    /** The start time of the meeting in ISO 8601 format. */
     startTime?: string;
+    /** The end time of the meeting in ISO 8601 format. */
     endTime?: string;
+    /** The subject line of the meeting. */
     subject?: string;
+    /** The body content of the meeting. */
```

Comment:
>The body content of the meeting [](http://example.com/codeflow?start=8&length=31)

What formats are supported for this content? HTML? Plain text?

Reply:
Content is not restricted to any specific format for the `ComposeMeetingParams.content` field. You can choose any format that fits developers use case, including plain text or HTML.